compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -36,11 +40,11 @@ pub(crate) struct Interpreter<'interner> {
 
     /// True if we've expanded any macros into any functions and will need
     /// to redo name resolution & type checking for that function.
-    changed_functions: HashSet<FuncId>,
+    pub(super) changed_functions: HashSet<FuncId>,
 
     /// True if we've expanded any macros into global scope and will need
     /// to redo name resolution & type checking for everything.
-    changed_globally: bool,
+    pub(super) changed_globally: bool,
```

Comment:
The intent for this was to be used just by `comptime::scan` since the interpreter impls are split across a couple files. I suppose I can move the scan module to a submodule of the interpreter to keep the fields private. They're not actually set at all yet currently though so I'll leave that for a later PR and revert this change for now.